drivers/ethernet/Kconfig.w5500

```diff
@@ -35,3 +35,13 @@ config ETH_W5500_TIMEOUT
 	  Given timeout in milliseconds. Maximum amount of time
 	  that the driver will wait from the IP stack to get
 	  a memory buffer before the Ethernet frame is dropped.
+
+config ETH_W5500_INT_TIMEOUT
+	int "Interrupt gpio timeout"
+	depends on ETH_W5500
+	default 100
+	help
+	  Maximum amount of time in milliseconds that the driver
+		will wait for an interrupt before manually checking
+		for incoming data. This adjusts how long before the driver
+		will recover from a missed interrupt.
```

Comment:
Indentation seems to be off for the last three lines (could be also my browser that shows these wrongly)